core/src/test/scala/unit/kafka/tools/StorageToolTest.scala

```diff
@@ -162,7 +163,7 @@ Found problem:
       val stream = new ByteArrayOutputStream()
       assertEquals(0, StorageTool.
         formatCommand(new PrintStream(stream), Seq(tempDir.toString), metaProperties, MetadataVersion.latest(), ignoreFormatted = false))
-      assertEquals("Formatting %s%n".format(tempDir), stream.toString())
+      assertTrue(stream.toString().startsWith("Formatting %s".format(tempDir)))
```

Comment:
Minor but you can use Scala string interpolation. E.g. `s"Formatting $tempDir"`.